loadflow/loadflow-api/src/main/java/com/powsybl/loadflow/json/LoadFlowResultDeserializer.java

```diff
@@ -72,7 +80,10 @@
             }
         }
 
-        if (componentNum == null) {
+        if (connectedComponentNum == null) {
+            connectedComponentNum = 0;
+        }
```

Comment:
That should be only for 1.1 version; for above versions I think we should raise an exception?